app/classifier/tasks/slider/index.jsx

```diff
@@ -37,94 +37,68 @@ SliderSummary.propTypes = {
   ).isRequired
 };
 
-class SliderTask extends React.Component {
-  constructor(props) {
-    super(props);
-    let value = this.props.task.defaultValue;
-    if (this.props.annotation.value !== null) {
-      value = this.props.annotation.value;
-    }
-    this.state = { value };
-    this.handleChange = this.handleChange.bind(this);
-  }
-
-  componentDidMount() {
-    const newAnnotation = Object.assign({}, this.props.annotation, { value: this.props.task.defaultValue });
-    this.props.onChange(newAnnotation);
-  }
-
-  componentWillReceiveProps(nextProps) {
-    if (this.props.annotation.value === null) {
-      const newAnnotation = Object.assign({}, this.props.annotation, { value: this.props.task.defaultValue });
-      this.props.onChange(newAnnotation);
-    } else if (this.props.annotation.value !== this.state.value) {
-      this.setState({ value: this.props.annotation.value });
-    }
-  }
-
-  handleChange(e) {
+const SliderTask = ({ task, annotation, onChange, autoFocus }) => {
+  const handleChange = (e) => {
```

Comment:
Not a blocking issue, but I find explicit function declarations `function handleChange(e) {…}` easier to read, and debug, than assigning an anonymous function to a variable.